kitsune/wiki/jinja2/wiki/revision.html

```diff
@@ -91,11 +91,5 @@ <h1 class="sumo-page-heading">{{ document.title }}</h1>
 {% endblock %}
 
 {% block side_top %}
-  {{ document_tools(document, document.parent, user, '', settings) }}
-{% endblock %}
-
-{% block side_bottom %}
```

Comment:
To be honest I would prefer to keep `show_for`instead of hiding the same call (with a different arg) a level deep. I would even go a step further and say that we should remove the `show_for` option from the doucument_tools and just use it directly. 

I don't object to passing a header of False but at the same time these pages are used heavily by contributors. I do have a concern that we will confuse users by removing just `Article is for:` without adding much to the aesthetics or the UX of the page.